syntax = "proto3";

service Showtime {
  rpc GetShowtimeByDate(Date) returns (ShowtimeData) {}
  rpc GetShowtimes(ShowtimeEmpty) returns (stream ShowtimeData) {}
}

message Date {
  string date = 1;
}

message ShowtimeData {
  string date = 1;
  repeated string movies = 2;
}

message ShowtimeEmpty {}
